# 実践問題19

In [3]:
from scipy import stats

In [4]:
X = [76, 82, 96, 70, 69]
Y = [42, 53, 81, 60, 65, 44]

In [5]:
stats.ttest_ind(X, Y)

Ttest_indResult(statistic=2.659781258383769, pvalue=0.026055134466048284)

検定統計量Tの実現値t=2.660は棄却域R(pvalue<0.005)に入らない。
よって、仮説H0:μx=μyは棄却されない。